packages/ketchup/src/f-components/f-text-field/f-text-field.tsx

```diff
@@ -452,24 +479,56 @@ function setHelper(props: FTextFieldProps): HTMLDivElement {
     }
 }
 
-const formatValue = function (
+function getMaximumNumbersLength(
```

Comment:
These functions are similar to those of the dates. Move the logic into the kup- math. By the way, the kup-math can already support these methods. Please check it out.

Reply:
moved functions in kup-math